src/ExpressionEvaluator/Core/Source/ResultProvider/SystemHelpers.cs

```diff
@@ -7,7 +7,13 @@ namespace System.Diagnostics
     internal static class Debug
     {
         [Conditional("DEBUG")]
-        public static void Assert(bool condition, string message = null)
+        public static void Assert(bool condition)
```

Comment:
Why did you enable nullable in this file? Should these get `DoesNotReturnIf(false)]`?

Reply:
I was forced to update the file to address a build error. I just didn't get around up to updating it for nullable yet.

Reply:
➡️ I'll leave this one out for now. There's always more work to do...